### PR TITLE
Logs now show dest. url and refactored auth builder logging

### PR DESCRIPTION
**Pull request by tschia**
also fixed typos

New log entry looks like:
Route - TransactionId: 55261316-080b-425c-957a-94a646724051 Message created successfully, sending to https://hnsesb-dev.apps.silver.devops.gov.bc.ca/hl7v2

### CODE REVIEW

File: hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Route.java [hunk 1]
```diff
@@ -107,23 +108,23 @@ public void configure() throws Exception {
         init();
 
         onException(HttpHostConnectException.class, UnknownHostException.class).process(new FailureProcessor())
-        .log("Recieved body ${body}").handled(true);
+        .log("Received body ${body}").handled(true);
```
Comment (tschia) [inline hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Route.java:111]: Typos

File: hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Route.java [hunk 2]
```diff
@@ -107,23 +108,23 @@ public void configure() throws Exception {
...
         onException(CamelCustomException.class).process(new FailureProcessor())
-        .log("Recieved body ${body}").handled(true);
+        .log("Received body ${body}").handled(true);
         
         from("direct:start").routeId("hnclient-route")
-            .log("Retrieving access token")
             .setHeader("Authorization").method(retrieveAccessToken).id("RetrieveAccessToken")
             .setBody().method(new Base64Encoder()).id("Base64Encoder")
             .setBody().method(new ProcessV2ToJson()).id("ProcessV2ToJson")
+            .log(LoggingLevel.INFO, logger,
```
Comment (tschia) [inline hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Route.java:123]: New logging to display sending destination

File: hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/authentication/ClientIdSecretBuilder.java [hunk 3]
```diff
@@ -38,8 +41,10 @@ public ClientIdSecretBuilder(String clientId, String clientSecret) {
     }
 
     @Override
-    public ClientAuthentication build() {
-        logger.info("Building client ID and secret.");
+    public ClientAuthentication buildAuthenticationMethod() {
+        String methodName = LoggingUtil.getMethodName();
+
+        logger.info("{} - Using ClientId and Secret for Authentication",methodName);
```
Comment (tschia) [inline hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/authentication/ClientIdSecretBuilder.java:47]: Made this logging consistent with our standard and updated the method name to be meaningful

**Review by daveb-hni — APPROVED** [review]
Logging now has URL and code refactor/cleanup looks good.

**Pull request merged**